docs/node-developers/graphql-api.mdx

```diff
@@ -143,14 +143,15 @@ subscription {
 }
 ```
 
-The new block subscription may also be limited to only return new blocks created by a defined public key with the `publicKey` argument.
+The new block subscription can also be limited to return only new blocks created by a defined public key with the `publicKey` argument.
 
 ### GraphQL Public Proxy
 
-While exposing the GraphQL API to the internet is not recommended, an implementation of a [GraphQL proxy](https://github.com/MinaProtocol/mina/tree/master/automation/services/graphql-public-proxy) is available that removes the queries that pose a threat but still allows querying of the node's data from a public endpoint.
+While exposing the GraphQL API to the internet is not recommended, an implementation of a [GraphQL proxy](https://github.com/MinaProtocol/mina/tree/master/automation/services/graphql-public-proxy) is available that removes the queries that pose a threat, but still allows querying of the node's data from a public endpoint.
 
 ### Resources
 
-- [5-minute introduction video on Mina GraphQL API](http://bit.ly/GraphQLAPIin5)
-- [First steps with the Mina GraphQL API](https://garethtdavies.com/crypto/first-steps-with-coda-graphql-api.html)
+- [GraphQL API Endpoint – Mina Explorer](https://www.youtube.com/watch?v=RKvg2t561J0)
+- [5-minute introduction video on Mina GraphQL API](http://bit.ly/GraphQLAPIin5) Coda
```

Comment:
I added Coda to the link just to get this line to show in the PR
- is the technical content in these two 2019 Coda videos still valid @psteckler ?

Reply:
mostly valid, but note that the Mina Explorer GraphQL API is separate from the GraphQL API exposed by the daemon; this document should make that distinction clear

Maybe this document should not even mention the Mina Explorer GraphQL API, since it's not provided by O(1)